packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -344,14 +344,30 @@ export class AtomicSearchBox implements InitializableComponent<Bindings> {
     | 'numberOfQueries'
     | 'clearFilters'
   > {
-    return {
-      ...this.bindings,
-      id: this.id,
-      isStandalone: () => !!this.redirectionUrl,
-      searchBoxController: () => this.searchBox,
-      numberOfQueries: this.numberOfQueries,
-      clearFilters: this.clearFilters,
-    };
+    return Object.defineProperties(
+      {...this.bindings},
```

Comment:
We're no longer returning id?

Reply:
Same comment for packages/atomic/src/components/commerce/atomic-commerce-search-box/atomic-commerce-search-box.tsx

Reply:
oops.